events/static/events/css/form.css

```diff
@@ -18,6 +18,10 @@
     border-radius: 5px;
 }
 
+.form > ul.errorlist{
+    display: none;
+}
+
```

Comment:
Hiding the built in djangos BaseForm error messaging since we've added our own

![image](https://user-images.githubusercontent.com/47626181/150420922-ec76d03d-bd13-4b88-9801-371a405103e6.png)